internal/types/utils/signals.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"syscall"
+
+	"github.com/hashicorp/go-version"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	sigNumSupportedVersion = version.Must(version.NewVersion("1.3.8"))
+	defaultSignals         = map[string]syscall.Signal{"DATA": syscall.SIGUSR1, "STATS": syscall.SIGUSR2}
+)
+
+func HasSigNumSupport(version *version.Version) bool {
+	return version == nil || version.GreaterThanOrEqual(sigNumSupportedVersion)
```

Comment:
Should we return true if the version is `nil`?

Reply:
This is the equivalent to this.
https://github.com/cafebazaar/keepalived-exporter/blob/7f70713c5b9d41121339e7e5195f91df9147fb1d/internal/collector/signal.go#L22-L25